packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx

```diff
@@ -242,6 +243,10 @@ const UserManagementPanel: FC = () => {
     const [showInviteModal, setShowInviteModal] = useState(false);
     const { data: organizationUsers, isLoading } = useOrganizationUsers();
 
+    if (user.data?.ability.cannot('view', 'OrganizationMemberProfile')) {
```

Comment:
I added this, this is the same condition we have to "show" the button on the sidebar. However, I think this is always true (for now) , even for viewers.